src/utils/path.js

```diff
@@ -48,7 +49,7 @@ export class ValueWrapper {
 
   constructor (value, curPath) {
     this.value = value
-    this.path = ValueWrapper.pathAsArray(curPath)
+    this.path = ValueWrapper.pathAsArray(parseVariables(this.value, curPath))
```

Comment:
How are you handling the exception that is thrown when `curPath` is empty or undefined. This can happen quite often in the form value, especially when you start off with an empty form value.